<launch>
  <node pkg="usv_map"
          name="geotf_server"
          type="geotf_server_node"
          respawn="false"
          output="screen">
  </node>
    
</launch>
